tools/python/find_optimizer_opset_version_updates_required.py

```diff
@@ -29,7 +29,7 @@ def parse_args():
     return args
 
 
-def get_call_args_from_file(filename, function_or_declaration):
+def get_call_args_from_file(filename: str, function_or_declaration: str) -> [str]:
```

Comment:
```suggestion
def get_call_args_from_file(filename: str, function_or_declaration: str) -> List[str]:
```

I wish the type were as nice as `[str]` though :)

Reply:
We need `from typing import List`